### DISCUSSION TITLE
域名证书过期,连不上,错误{"code":"CERT_HAS_EXPIRED"}

### DISCUSSION THREAD

**Discussion by leehom66** [antoor/antSword#327] [Q&A] [answered]
遇到了 证书过期的域名,提示错误{"code":"CERT_HAS_EXPIRED"}

1.在默认设置 忽略HTTPS证书 勾选，重启后 无效
2.搜索了全网,按步骤,发现 文章https://blog.csdn.net/Adminxe/article/details/108744919
但是解决的是2.0之前的版本,现在并无jar文件,所以也无法 运行java -Djsse.enableSNIExtension=false -jar xxxx.jar

希望能兼容这种 过期证书的域名

**Comment by Medicean** [comment, marked as answer]
https://www.yuque.com/antswordproject/antsword/iunkur#87ELa
